Gemfile

```diff
@@ -5,6 +5,7 @@ gemspec
 group :development, :test do
   gem 'prettier'
   gem 'rake'
+  gem 'rbs', '< 3.2' # drops support for ruby 2.7, which we still need
```

Comment:
If we go for `ruby 2.7.0` in tool-versions, wouldn't this resolve, and then when we update it there, we'll get the upgrade to rbs 3.2+ automatically?

Reply:
cc @brerx what do you think if we do this like here https://github.com/ivx/ears/pull/31?